docs/architecture/payments.rst

```diff
@@ -1,6 +1,48 @@
 Payments
 ========
 
+Supported Gateways
+------------------
+
+The default configuration uses the *dummy* backend.
+It's meant to allow developers to easily simulate different payment results.
+
+Here is a list of supported payment providers:
+
+* Braintree
+
+For how-to guide on adding new payments into your Saleor project please check :ref:`adding-payments`
+
+.. note::
+
+    All payment backends default to using sandbox mode.
+    This is very useful for development but make sure you use production mode when deploying to a production server.
+
+
+Authorization and Capture
+-------------------------
+
+Some of the payment backends support pre-authorizing payments.
+
+Authorization and capture is a two-step process.
+
+Firstly the funds are locked on the payer's account but are not transferred to your bank.
+
+Then depending on the provider and card type you have between a few days and a month to charge the card for an amount not exceeding the authorized amount.
```

Comment:
Would suggest this:
Then depending on the provider and **the** card type*,* you have between a few days*,* and a month